### PR TITLE
Plugin Graphs: make history file configurable

### LINKED ISSUE #517: HISTORY_DUMP from config
Hello,

HISTORY_DUMP = "www/history.json" is defined static in code. And when I'm changing the name of www folder, the following error is given:

Short higher: 2017-09-17 14:10:52 Error 
Traceback (most recent call last):
  File "/opt/poloniexlendingbot/lendingbot.py", line 95, in <module>
    PluginsManager.after_lending()
  File "/opt/poloniexlendingbot/modules/PluginsManager.py", line 47, in after_lending
    plugin.after_lending()
  File "/opt/poloniexlendingbot/plugins/Charts.py", line 33, in after_lending
    self.dump_history()
  File "/opt/poloniexlendingbot/plugins/Charts.py", line 63, in dump_history
    with open(HISTORY_DUMP, "w") as hist:
IOError: [Errno 2] No such file or directory: 'www/history.json'

Could you please add "historyfile" to config file like it is done for jsonfile? 

jsonfile = www-myown/botlog.json

Thanks!

Best regards,

Thomas

### PR DESCRIPTION
**Pull request by utdrmac**
Resolves #517

### CODE REVIEW

File: default.cfg.example [hunk 1]
```diff
@@ -132,7 +132,8 @@ hideCoins = True
 #ReportInterval = 86400
 
 #[CHARTS]
-#DumpInterval=21600
+#DumpInterval = 21600
+#HistoryFile = www/history.json
```
Comment (rnevet) [inline default.cfg.example]: Naming convention, lower cased + underscore

Comment (utdrmac) [reply default.cfg.example]: What specifically? I was going off AccountStats variable ReportInterval convention.

**Comment by utdrmac** [comment]
@MoonMaker I've been using this:
```
cat $HOME/.gitconfig
[alias]
  pr = "!f() { git fetch -fu ${2:-upstream} refs/pull/$1/head:pr/$1 && git checkout pr/$1; }; f"
```

Allows for `git pr 518` and checks out the PR as a local branch

File: plugins/Charts.py [hunk 2]
```diff
@@ -22,6 +21,7 @@ def on_bot_init(self):
         self.db = sqlite3.connect('market_data/loan_history.sqlite3')
         self.last_dump = 0
         self.dump_interval = int(self.config.get("CHARTS", "DumpInterval", 21600))
+        self.history_file = self.config.get("CHARTS", "HistoryFile", "www/history.json"))
```
Comment (MoonMaker) [inline plugins/Charts.py]: Please remove the last ")" sign. After this change it is working very well.

File: default.cfg.example (hunk 1 above)
Comment (rnevet) [reply default.cfg.example]: Well, you also introduced 'ReportInterval'  and actually they all should be lowercase. :)

**Pull request merged**